web-app/src/components/Navbar.js

```diff
@@ -32,6 +32,33 @@ const Navbar = () => {
 
 const StyledNav = styled.nav`
   display: flex;
+
+  @media (max-width: 576px) {
+    width: 100%;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: white;
+    transform: ${({ open }) => (open ? "translateX(0)" : "translateX(-100%)")};
+    height: 100vh;
+    text-align: left;
+    padding: 2rem;
+    position: absolute;
+    top: 0;
+    left: 0;
+    transition: transform 0.3s ease-in-out;
+
```

Comment:
Position needs to be set to fixed. In mobile, when you open the Navbar, you can scroll down past it.

Reply:
try again and let me know